builtin/credential/github/backend.go

```diff
@@ -61,6 +75,20 @@ func Backend() *backend {
 		OperationPrefix: operationPrefixGithub,
 		OperationSuffix: "user-mapping",
 	}
+	userMapPaths[0].Operations = map[logical.Operation]framework.OperationHandler{
+		logical.ListOperation: &framework.PathOperation{
+			Callback: userMapPaths[0].Callbacks[logical.ListOperation],
+			Summary:  userMapPaths[0].HelpSynopsis,
+		},
+		logical.ReadOperation: &framework.PathOperation{
+			Callback: userMapPaths[0].Callbacks[logical.ReadOperation],
+			Summary:  userMapPaths[0].HelpSynopsis,
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationSuffix: "users2", // The ReadOperation is redundant with the ListOperation
```

Comment:
```suggestion
				OperationVerb:   "list",
				OperationSuffix: "users2", // The ReadOperation is redundant with the ListOperation
```
This will result in `github-list-users2` instead of `github-read-users2`